src/createElement.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import StemContainer from './StemContainer';
```

Comment:
What do you think about `RouteContainer` or `RouteComponentContainer`? Or just `Container`. Not sure about `RootContainer` since we're rendering one of these around route component.